platformio.ini

```diff
@@ -0,0 +1,22 @@
+; PlatformIO Project Configuration File
+;
+;   Build options: build flags, source filter
+;   Upload options: custom upload port, speed and extra flags
+;   Library options: dependencies, extra library storages
+;   Advanced options: extra scripting
+;
+; Please visit documentation for the other options and examples
+; https://docs.platformio.org/page/projectconf.html
+
+[env:esp32dev]
+platform = espressif32
+board = esp32dev
+framework = arduino
+monitor_speed = 115200
+;upload_protocol = espota
+;upload_port = 192.168.2.199
+upload_port  = /dev/ttyUSB0
+monitor_port = /dev/ttyUSB0
```

Comment:
These entries in the .ini seem like they're specific to your environment, so I don't think we could include them in the regular repository (on Windows it may be COM# or on MacOS `/dev/tty.usbserial`), and then it depends which device number is assigned.